infra/README.md

```diff
@@ -44,15 +44,17 @@ Functional examples are included in the
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
+| client\_image\_host | Container Registry that hosts the client image (PROJECT\_ID[/folder]) | `string` | `"hsa-public/terraform-python-dynamic-webapp"` | no |
```

Comment:
question: Some of these changes seem like unrelated readme docs regeneration. Should there be a separate process to make that happen?

Reply:
Looking into this more: Normally yes, but there was an issue with the linter that was fixed in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1484

https://console.cloud.google.com/cloud-build/builds/94a0ce8b-8629-41bc-be5c-1953d7d9a164?project=2843445864 (from earlier in this PR) successfully showed it failed.

https://console.cloud.google.com/cloud-build/builds/1a81afbd-3fbc-42f8-befe-6fbbd027494b?project=2843445864 (from #37) failed, but the error code was incorrect so the status was 'success'. 

Since the linting was failing on my PR, I fixed them. I could split them out into a different PR, but then this PR would continue to fail until that PR was merged and this rebased.